sqlglot/dialects/singlestore.py

```diff
@@ -57,6 +57,7 @@ class Tokenizer(MySQL.Tokenizer):
             "!:>": TokenType.NCOLON_GT,
             "::$": TokenType.DCOLONDOLLAR,
             "::%": TokenType.DCOLONPERCENT,
+            "REGEXP": TokenType.RLIKE,
```

Comment:
This is redundant, it's [included](https://github.com/tobymao/sqlglot/blob/e71bcb51181de63c8ad13004216506529fcf9644/sqlglot/tokens.py#L829) in the base tokenizer.
```suggestion
```